src/services/concretes/socketIO/socketIO.service.ts

```diff
@@ -0,0 +1,47 @@
+import { listen, Socket } from "socket.io";
+import { connect } from "socket.io-client";
+import { MessageType } from "../../../enums/messageType.enum";
+import { IMessageListener } from "../../../interfaces/messageListener";
+import { Message } from "../../../models/message.model";
+import { ICommunicationProtocol } from "../../interfaces/communicationProtocol.service";
+import { SocketIOReceiver } from "./socketIOReceiver.service";
+import { SocketIOSender } from "./socketIOSender.service";
+
+/**
+ * Handles the sending of messages.
+ */
+export class SocketIOCommunicationProtocol implements ICommunicationProtocol {
+    private readonly sender: SocketIOSender;
+    private readonly reciever: SocketIOReceiver;
+
```

Comment:
Please remove this duplicate newline

Reply:
Please change the `reciever` typo to `receiver`.